docker-compose.yaml

```diff
@@ -35,7 +35,7 @@ services:
     labels:
       - "com.centurylinklabs.watchtower.enable=true"
     ports:
-      - "10003:10003"      
```

Comment:
Can u undo this change

Reply:
Good highlight, @nikita15p. Ideally, this change is reverted, but if doing so is a challenge, @ratatatCat, just signal. It's an inconsequential change and so, ultimately, we can move forward even with it here.

Reply:
Fixed!